activesupport/lib/active_support/core_ext/object/blank.rb

```diff
@@ -117,6 +117,10 @@ class String
   #
   #   "\u00a0".blank? # => true
   #
+  # Unicode zero width space is supported:
```

Comment:
We should merge those examples with the one above.

```suggestion
  # Unicode whitespace and zero width space are supported:
```

Reply:
done